### DISCUSSION TITLE
Need help getting gettouch to work with an 8 bit parallel ssd 1963 from buydisplay.com

### DISCUSSION THREAD

**Discussion by dunn-michael** [Bodmer/TFT_eSPI#2874] [Q&A - Touch]
Hi we are trying to get squareline studio and LVGL to work with our code but we cannot get the tft_espi library to work because we are using a display that uses 8-bit parallel. Does anybody have any work arounds so that we can get our screen to work with tft_espi get touch function. If you have any recommended libraries that work well please refer them to me! If you can give the code that you used to work that would be great.

We are using arduino and a ESP32.

**Comment by Bodmer** [comment]
As you have found, TFT_eSPI doesn't support touch with a parallel  interface.  For the capacitive screen option of that display I created a library here:
https://github.com/Bodmer/Touch_FT5436
For other touch device controllers an alternative library will be required.

**Reply by dunn-michael** [reply]
That link didn't take me anywhere

**Reply by Bodmer** [reply]
Try again, it was private, now public.
